### PR TITLE
fix(heatmap): add detail descriptions for heatmap 'normalize across'

### PR DESCRIPTION
**Pull request by stephenLYZ**
### SUMMARY

Add detail desciptions for heatmap 'normalize across'
- y -> percentage of the highest value in that row
- x -> percentage of the highest value in that column
- heatmap -> percentage of the highest value in the whole heatmap
### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/plugins/legacy-plugin-chart-heatmap/src/controlPanel.tsx [hunk 1]
```diff
@@ -163,17 +164,34 @@ const config: ControlPanelConfig = {
             name: 'normalize_across',
             config: {
               type: 'SelectControl',
-              label: t('Normalize Across'),
+              label: t('COLOR BASED ON'),
```
Comment (rusackas) [inline superset-frontend/plugins/legacy-plugin-chart-heatmap/src/controlPanel.tsx:167]:
```suggestion
              label: t('Color based on'),
```

**Comment by yousoph** [comment]
image
the dropdown options don't look renamed in the ephemeral env

**Comment by stephenLYZ** [comment]
@yousoph Yeah, the point I'm currently struggling with is that using like "percentage of the highest value in that row" as an option label is too long, can we simplify it? Or just re-label to "row" and add some tooltip.

File: superset-frontend/plugins/legacy-plugin-chart-heatmap/src/controlPanel.tsx [hunk 2]
```diff
@@ -163,17 +164,34 @@ const config: ControlPanelConfig = {
...
               default: 'heatmap',
-              description: t(
-                'Color will be rendered based on a ratio ' +
-                  'of the cell against the sum of across this ' +
-                  'criteria',
+              description: (
+                <>
+                  <div>
+                    {t(
+                      'Color will be rendered based on a ratio of the cell against the sum of across this criteria: ',
```
Comment (rusackas) [inline superset-frontend/plugins/legacy-plugin-chart-heatmap/src/controlPanel.tsx:178]:
```suggestion
                      'Color will be shaded based the normalized value of a given cell against the other cells in the selected range: ',
```

Comment (rusackas) [reply superset-frontend/plugins/legacy-plugin-chart-heatmap/src/controlPanel.tsx:178]: @yousoph does this make more sense?

File: superset-frontend/plugins/legacy-plugin-chart-heatmap/src/controlPanel.tsx [hunk 3]
```diff
@@ -163,17 +164,34 @@ const config: ControlPanelConfig = {
...
+                    )}
+                  </div>
+                  <ul>
+                    <li>
+                      {t('x: percentage of the highest value in that column')}
```
Comment (rusackas) [inline superset-frontend/plugins/legacy-plugin-chart-heatmap/src/controlPanel.tsx:183]:
```suggestion
                      {t('x: values are normalized within each column')}
```

File: superset-frontend/plugins/legacy-plugin-chart-heatmap/src/controlPanel.tsx [hunk 4]
```diff
@@ -163,17 +164,34 @@ const config: ControlPanelConfig = {
...
+                    </li>
+                    <li>
+                      {t('y: percentage of the highest value in that row')}
```
Comment (rusackas) [inline superset-frontend/plugins/legacy-plugin-chart-heatmap/src/controlPanel.tsx:186]:
```suggestion
                      {t('y: values are normalized within each row')}
```

File: superset-frontend/plugins/legacy-plugin-chart-heatmap/src/controlPanel.tsx [hunk 5]
```diff
@@ -163,17 +164,34 @@ const config: ControlPanelConfig = {
...
+                    </li>
+                    <li>
+                      {t(
+                        'heatmap: percentage of the highest value in the whole heatmap',
```
Comment (rusackas) [inline superset-frontend/plugins/legacy-plugin-chart-heatmap/src/controlPanel.tsx:190]:
```suggestion
                        'heatmap: values are normalized across the entire heatmap',
```

File: superset-frontend/plugins/legacy-plugin-chart-heatmap/src/controlPanel.tsx [hunk 6]
```diff
@@ -163,17 +164,34 @@ const config: ControlPanelConfig = {
             name: 'normalize_across',
             config: {
               type: 'SelectControl',
-              label: t('Normalize Across'),
+              label: t('Color based on'),
```
Comment (rusackas) [inline superset-frontend/plugins/legacy-plugin-chart-heatmap/src/controlPanel.tsx:167]: I think I might be OK with "Normalize Across" now that we know it's actually doing normalization

File: superset-frontend/plugins/legacy-plugin-chart-heatmap/src/controlPanel.tsx (hunk 2 above)
Comment (yousoph) [reply superset-frontend/plugins/legacy-plugin-chart-heatmap/src/controlPanel.tsx:178]: Yes!

**Review by rusackas — APPROVED** [review]
LGTM. This just needs a rebase to make codecov happy. @yousoph let me know if you agree on the assessment - I think this is clearer.

**Pull request merged**